openedx/features/partners/give2asia/views.py

```diff
@@ -110,25 +104,16 @@ def create_account_with_params_custom(request, params, partner):
             log.exception("User creation failed for user {username}".format(username=params['username']), repr(err))
             raise
 
-        try:
-            # Update User profile
-            profile_fields = [
-                "name", "level_of_education", "gender", "country", "year_of_birth"
-            ]
-            profile = UserProfile(
-                user=user,
-                **{key: params.get(key) for key in profile_fields}
-            )
-
-            # Return a dictionary containing the extended_profile_fields and values
-            extended_profile_data = {
-                key: value
-                for key, value in params.items() if key in extended_profile_fields and value is not None
-            }
+        extended_profile_fields = g2a_constants.G2A_EXTENDED_PROFILE_DEFAULT_DATA
```

Comment:
Let's please rename `extended_profile_fields` because it contains data not fields